library/bind.c

```diff
@@ -405,8 +405,8 @@ static void bind_reply_cb(void *ctx, message *msg, int code) {
     struct ziti_conn *conn = b->conn;
 
     b->waiter = NULL;
-    CONN_LOG(TRACE, "received msg ct[%X] code[%d]", msg->header.content, code);
     if (code == ZITI_OK && msg->header.content == ContentTypeStateConnected) {
```

Comment:
can `msg` be null when code == `ZITI_OK`?

Reply:
no